### PR TITLE
CUDOS-756: Add "connect signer" and "private accounts"

### PR DESCRIPTION
**Pull request by aemil145**
https://cudo-ventures.atlassian.net/browse/CUDOS-756

- User will be able to connect an instance of a contract to a different signer
- User will be able to use his own private accounts preventing himself from committing them
- Optimized exposed global functions and contract functions
- Updated documentation and added description of contract functions

### CODE REVIEW

File: packages/blast-utilities/contract-utils.js [hunk 1]
```diff
@@ -9,29 +9,25 @@ const BlastError = require('./blast-error')
 const { getProjectRootPath } = require('./package-info')
 
 module.exports.CudosContract = class CudosContract {
-  #contractName
+  #contractLabel
```
Comment (maptuhec) [inline packages/blast-utilities/contract-utils.js:12]: why is ti changed from name to label?

File: template/private-accounts.json [hunk 2]
```diff
@@ -1,8 +1,6 @@
 {
-    "comment": "It is highly recommended to .gitignore this file",
-    "privateAccount":
-    {
-      "address": "",
-      "mnemonic": ""
-    }
+	"comment": "It is highly recommended to .gitignore this file",
```
Comment (maptuhec) [inline template/private-accounts.json]: ca we directly add it to git ignore?

**Review by maptuhec — CHANGES_REQUESTED** [review]

File: template/private-accounts.json (hunk 2 above)
Comment (Holymir) [reply template/private-accounts.json]: if we add it to .gitignore we will ignore the file itself.

**Review by Holymir — APPROVED** [review]

File: packages/blast-utilities/contract-utils.js (hunk 1 above)
Comment (aemil145) [reply packages/blast-utilities/contract-utils.js:12]: I changed it for consistency with the "label" we are getting from CosmWasmClient in globals.js in the "getContractFromAddress" function where we get a contract object with a label from an already deployed contract

File: template/private-accounts.json (hunk 2 above)
Comment (aemil145) [reply template/private-accounts.json]: the idea is to add this file in template/gitignore so when the user inits a sample project, Cudos blast will copy gitignore and private-accounts folders, so in the new dir private-accounts will be automatically ignored

**Review by maptuhec — APPROVED** [review]

**Review by maptuhec — APPROVED** [review]

**Pull request merged**